OneCommunityGlobal/HighestGoodNetworkApp
# Jiayu took this over from Miguel - User Suggestion Edit

# Description
<img width="691" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/14044314/64b973bb-3c4f-4752-b5e7-7692d9f23792">


## Related PRS (if any):
N/A

## Main changes explained:
- Updated SummaryBar.jsx to accommodate new "Edit" radio button and logic
- Update SummaryBar.css to have specialized styles for Edit list.
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as Owner user. If you don't have one, I believe Jay can manually make one for you.
5. Under the dashboard, there should be five different icons at the top right side. Click on the lightbulb icon, "Suggestions Development"
6. Click on the Dropdown menu and verify the order. Example:
<img width="561" alt="image" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/14044314/425fa4c2-0117-4a6a-bbd3-eacb1554c8d2">

7. Click "Edit Category" button at top left of modal.
8. Click "Edit", a list of the dropdown options should appear as draggable boxes.
9. Click and drag on the white box to re-order them to any order you specifiy.
10. Click on the pencil-notepad icon at the top right of the white box to edit the text.
11. Click Submit to submit the changes to dropdown.

## Screenshots or videos of changes:

Video:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/14044314/9a2b5c53-64fe-470d-964c-cefc31b403a0



## Note:
